DaS-PC-MPChan/DarkSoulsProcess.vb

```diff
@@ -82,72 +86,63 @@ Public Class DarkSoulsProcess
         Dispose()
     End Sub
 
-    Private Function TryAttachToProcess(ByVal windowCaption As String) As Boolean
+    Private Sub attachToProcess()
+        Dim windowCaption As String = "darksouls"
```

Comment:
I just changed the parameter to a fixed local variable. Since the exceptions talk about Dark Souls having this as a parameter seemed wrong :). I didn't change the name or functionality.